magda-web-client/src/UI/Pagination.js

```diff
@@ -44,90 +44,85 @@ class Pagination extends Component {
     }
 
     renderPageList(max, current) {
```

Comment:
Can we rename these to `maxPage` and `currentPage` (I assume that's what they do?)